neptune/exceptions.py

```diff
@@ -71,32 +112,132 @@ def __init__(self, expected_type, actual_type):
                 expected=expected_type, actual=actual_type))
 
 
-class NoExperimentContext(NeptuneException):
+class NeptuneNoExperimentContextException(NeptuneException):
     def __init__(self):
-        super(NoExperimentContext, self).__init__('Unable to find current active experiment')
+        message = """
+{h1}  
+----NeptuneNoExperimentContextException---------------------------------------------------------------------------------
+{end}
+Neptune couldn't find an active experiment.
 
+Looks like you forgot to run:
+    {python}neptune.create_experiment(){end}
+    
+You may also want to check the following docs pages:
+    - https://docs.neptune.ai/getting-started/quick-starts/log_first_experiment.html
 
-class MissingApiToken(NeptuneException):
-    def __init__(self):
-        super(MissingApiToken, self).__init__('Missing API token. Use "{}" environment '
-                                              'variable or pass it as an argument to neptune.init. '
-                                              'Open this link to get your API token '
-                                              'https://ui.neptune.ai/get_my_api_token'
-                                              .format(envs.API_TOKEN_ENV_NAME))
+{correct}Need help?{end}-> https://docs.neptune.ai/getting-started/getting-help.html
+""".format(**STYLES)
+        super(NeptuneNoExperimentContextException, self).__init__(message)
 
 
-class MissingProjectQualifiedName(NeptuneException):
+class NeptuneMissingApiTokenException(NeptuneException):
     def __init__(self):
-        super(MissingProjectQualifiedName, self).__init__('Missing project qualified name. Use "{}" environment '
-                                                          'variable or pass it as an argument'
-                                                          .format(envs.PROJECT_ENV_NAME))
+        message = """
+{h1}
+----NeptuneMissingApiTokenException-------------------------------------------------------------------------------------
+{end}
+Neptune client couldn't find your API token.
+
+Learn how to get it in this docs page:
+https://docs.neptune.ai/security-and-privacy/api-tokens/how-to-find-and-set-neptune-api-token.html
+
+There are two options two add it:
+    - specify it in your code 
+    - set an environment variable in your operating system.
+
+{h2}CODE{end}
+Pass the token to {bold}neptune.init(){end} via {bold}api_token{end} argument:
+    {python}neptune.init(api_token='YOUR_LONG_TOKEN', project_qualified_name='WORKSPACE_NAME/PROJECT_NAME'){end}
```

Comment:
Same comments here as in `NeptuneUninitializedException`.

Reply:
Done